engine/src/exec-interface-graphic.cpp

```diff
@@ -713,6 +733,29 @@ void MCGraphic::GetMarkerPoints(MCExecContext& ctxt, uindex_t& r_count, MCPoint*
 
 void MCGraphic::SetMarkerPoints(MCExecContext& ctxt, uindex_t p_count, MCPoint* p_points)
 {
+    if (p_count > 65535)
```

Comment:
It should be possible to simplify this handler... The closepolygon() method does not mutate its arguments if it fails. So this method should be able to be split up into two halves - the first half computes the new points that it will set; if that succeeds (because closepolygon succeeds if the marker points are opaque), then commit new points.